include/seastar/parquet/overloaded.hh

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
Feel free to put this one in `include/seastar/util`, wrap it in a separate patch and send it to sestar-dev - we do have `overloaded` in scylla already, but seastar is eventually going to need it too.